conversion/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "babel": {
+    "plugins": ["transform-inline-environment-variables"],
+    "presets": [
+      [
+        "@babel/preset-env",
+        {
+          "targets": "cover 99.5%"
```

Comment:
Not sure how many browsers we want to support

Reply:
We'll follow the google standard.

Reply:
And that is?

Reply:
Updated this to what we discussed on Slack, see [this link](https://browserl.ist/?q=defaults%2C+ie+%3E+10) for the full list